src/cpp/cpp_instantiate_template.cpp

```diff
@@ -150,7 +150,7 @@ const symbolt &cpp_typecheckt::class_template_symbol(
   const cpp_template_args_tct &full_template_args)
 {
   // we should never get 'unassigned' here
```

Comment:
Can these comments also be removed? They duplicate the intent of the programmatic check.

Reply:
Done!